CODEOWNERS

```diff
@@ -1,6 +1,6 @@
 # These owners will be the default owners for everything in the repo.
-* @microsoft/dynamics-smb-engineering-systems
+* @microsoft/d365-bc-engineering-systems
 
 # Build system files
-/.github @microsoft/dynamics-smb-engineering-systems
-/CODEOWNERS @microsoft/dynamics-smb-engineering-systems
+/.github @microsoft/d365-bc-engineering-systems
```

Comment:
These could be removed - We're already owners of everything :)

Reply:
Ok, ok...